requirements.txt

```diff
@@ -12,13 +12,11 @@ Flask==1.0.2
 Flask-BabelEx==0.9.3
 Flask-Login==0.4.1
 Flask-Mail==0.9.1
-Flask-Principal==0.4.0
 Flask-Redis==0.3.0
 Flask-RESTful==0.3.6
-Flask-Security==3.0.0
 flask-sqlacodegen==1.1.6.1
 Flask-SQLAlchemy==2.3.2
-Flask-WTF==0.14.2
+gunicorn==19.9.0
```

Comment:
Nice, Was just looking into gunicorn and seems like a helpful module

Reply:
Yeah. It scales good and is easier to work configure.